README.md

```diff
@@ -102,27 +102,25 @@ CeresDB is a high-performance, distributed, schema-less, cloud native time-serie
 <dependency>
   <groupId>io.ceresdb</groupId>
   <artifactId>ceresdb-all</artifactId>
-  <version>0.1.0</version>
+  <version>1.0.0.alpha</version>
```

Comment:
Refer to CeresDB's versions, `1.0.0-alpha` will be better.
```suggestion
  <version>1.0.0-alpha</version>
```

Reply:
fixed